Analyse_Birds.qmd

```diff
@@ -436,11 +450,389 @@ for (i in 1:NSim) {
+
+# Dataset : variables as factors
+
+data$State<-as.factor(data$State)
+data$Habitat<-as.factor(data$Habitat)
+data$Species<-as.factor(data$Species)
+data$Year<-as.factor(data$Year)
+str(data)
+
+```
+
+```{r Ini_missing_data}
+#| echo: true
+#| code-fold: true
+#| code-summary: "Show the code"
+
+# Checking for missing data
+
+colSums(is.na(data))  
+```
+  
+There is no missing data here.
+  
+  
+Before any modeling, we must explore the data in order to avoid possible errors. 
+Below we present the list of steps to follow before proceeding to the modeling:
+
+- 1: Check for the presence of outliers in $Y$ and the distribution of the 
+values of $Y$
+
+- 2: If $X$ is a **quantitative** explanatory variable, check for the presence 
+of outliers and the distribution of the values of $X$ **OR/AND** 
+If $X$ is a **qualitative** explanatory variable, analyze the number of 
+modalities and the number of individuals per modality
+
+- 3: Analyze the potential relationship between $Y$ and the $X_{s}$
+
+- 4: Check for the presence of possible interactions between the $X_{s}$
+  
+  
+### Outliers in $Y$ and distribution of values of $Y$
+
+```{r Ini_datahist, fig.height=5, fig.width=5}
+#| echo: true
+#| code-fold: true
+#| code-summary: "Show the code"
+par(mfrow=c(2,2))
+# Boxplot
+boxplot(data$Initiation_Date,col='dodgerblue3',ylab='Initiation date')
+# Cleveland plot
+dotchart(data$Initiation_Date,pch=16,col='dodgerblue3',xlab='Initiation date')
+# Histogram
+hist(data$Initiation_Date,col='dodgerblue3',xlab="Initiation date",main="")
+# Quantile-Quantile plot
+qqnorm(data$Initiation_Date,pch=16,col='dodgerblue3',xlab='')
+qqline(data$Initiation_Date,col='red',lwd=2)
+```
+  
+  
+The different graphs show us no outliers for Initiation_Date. In addition, our 
+Y variable seems to follow a Normal Distribution.
+  
+  
+  
+### Outliers and distribution of the values of $X$
+
+The four $Xs$ are factors. We can analyze the number of modalities and the number of 
+individuals per modality. 
+
+```{r Ini_datafact}
+#| echo: true
+#| code-fold: true
+#| code-summary: "Show the code"
+
+# Factor State
+summary(data$State)
+# Factor Habitat
+summary(data$Habitat)
+# Factor Species
+summary(data$Species)
+# Factor Year
+summary(data$Year)
+```
+
+Factor State: The experimental design is not balanced (1071 and 2150 data), but
+the analysis can be performed because there are enough data in each modality (> 10).
+
+Factor Habitat: The experimental design is not balanced (from 392 to 1683 data), but
+the analysis can be performed because there are enough data in each modality (> 10).
+
+Factor Species: The experimental design is not balanced. Several species have less than 10
+data per modality. It will therefore be necessary to remove these data before continuing
+the analysis
+
+Factor Year: The experimental design is not balanced (from 24 to 448 data), but
+the analysis can be performed because there are enough data in each modality (> 10).
+
```

Comment:
Do we precise why we choose the >10 threshold?

Reply:
It's the threshold from which we consider that carrying out statistical analyses makes sense. But each scientist can choose his own threshold. The explanations have been provided in the document.